### PR TITLE
feature: add docker postgres-db

### PR DESCRIPTION
**Pull request by JoeSchr**
can be run by simply typing `docker-compose up`

### CODE REVIEW

**Comment by JoeSchr** [comment]
should address: https://github.com/jsappme/node-binance-trader/issues/53

File: backtest.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ const { Client } = require('pg')
 
 //////////////////////////////////////////////////////////////////////////////////
 
-const test_pair = "BTCUSDT"
```
Comment (dargmuesli) [inline backtest.js:9]: Why this change? Is it related to this PR?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+      - POSTGRES_USER=${POSTGRES_USER:-postgres}
+      - POSTGRES_DB=${POSTGRES_DB:-postgres}
+    volumes:
+      - db-volume:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    networks:
+      - default
```
Comment (dargmuesli) [inline docker-compose.yml:16]: Is a default network really required if not for a particular use case?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    command: >
+      postgres -c logging_collector=on -c log_destination=stderr -c
+      log_directory=/var/lib/postgresql/data/logs -c log_rotation_age=60 -c
+      log_truncate_on_rotation=on -c log_filename=server_log.hour.%H%M
```
Comment (dargmuesli) [inline docker-compose.yml:20]: Also, why this command? :)

Comment (JoeSchr) [reply docker-compose.yml:20]: this command I just copy and pasted from [another project (a nuxtjs variant of graphile/starter)](https://github.com/JoeSchr/graphile-starter) I'm using a postgres db docker-compose.yml as well. I figured it didn't hurt me, so why not use it. Honestly I think I just got this from upstream...

File: docker-compose.yml (hunk 2 above)
Comment (JoeSchr) [reply docker-compose.yml:16]: I don't think so

File: backtest.js (hunk 1 above)
Comment (JoeSchr) [reply backtest.js:9]: nope

**Comment by JoeSchr** [comment]
@dargmuesli  cleaned it up. thanks for reviewing it!

File: docker-compose.yml (hunk 3 above)
Comment (dargmuesli) [reply docker-compose.yml]: Really cool to see someone in the crypto space who also works with graphile :)

**Comment by JoeSchr** [comment]
haha, I'm in crypto since last week! :D

you use postgraphile too? with which frontend?


On Sun, 31 Jan 2021 22:32:16 +0100, Jonas Thelemann
<notifications@github.com> said:

**Comment by dargmuesli** [comment]
> haha, I'm in crypto since last week! :D

for me it's about a year, but bots since last week too 😄

> you use postgraphile too? with which frontend?

Nuxt.js, I'm working on maev.si (https://github.com/maevsi/maevsi), an event managing tool.
